src/main/java/com/google/sps/servlets/GoServlet.java

```diff
@@ -148,7 +148,7 @@ public void doGet(
     List<Task> tasks;
 
     String taskLists = request.getParameter("taskLists");
-    if (taskLists == null) {
+    if (taskLists.isEmpty()) {
```

Comment:
I am not sure what prompted this change, but you may need to handle both null and empty here?

For future reference, the Apache Commons package has a utility method called `StringUtils.isEmpty` that handles `null` the same as empty.

Reply:
I found that because `taskLists=` will always be in the query string, getting the `taskLists` parameter will return `""` instead of `null`. However, I changed it to `StringUtils.isEmpty` anyway just so I'll remember it exists next time. Thanks!

Reply:
`RESOLVED`